apps/dashboard/test/system/jobs_app_test.rb

```diff
@@ -64,7 +64,7 @@ def add_account(project_id, script_id)
   def add_bc_num_hours(project_id, script_id)
     visit project_path(project_id)
     edit_script_path = edit_project_script_path(project_id, script_id)
-    find(%Q{[href="#{edit_script_path}"]}).click
+    find("[href='#{edit_script_path}']").click
```

Comment:
@abujeda FYI I'm changing this. Couldn't put my finger on it in the inital PR - but I knew that %Q syntax looked odd.